### PR TITLE
Optimize lifetime_value method

### PR DESCRIPTION
**Pull request by jitendra**
use arel sum instead of pluck and ruby sum

### CODE REVIEW

**Comment by priyank-gupta** [comment]
@nishant-cyro This has failing specs in user model. Please fix.

**Comment by nishant-cyro** [comment]
@priyank-gupta - Fixed the specs.

File: core/.byebug_history [hunk 1]
```diff
@@ -0,0 +1,34 @@
+c
```
Comment (priyank-gupta) [inline core/.byebug_history]: @nishant-cyro I think this file is not intended to be here.

Comment (nishant-cyro) [inline core/.byebug_history]: @priyank-gupta - yeah, removed :+1:

**Comment by priyank-gupta** [comment]
:+1: 
@damianlegawiec @Mafi88 ?

**Comment by damianlegawiec** [comment]
@nishant-cyro - please do rebase & squash :)

**Comment by nishant-cyro** [comment]
@damianlegawiec Rebased! Please rebuild specs.

**Comment by Mafi88** [comment]
:+1:

**Pull request merged**